test/functional/test-xhr.js

```diff
@@ -341,19 +341,23 @@ describe('XHR', function() {
       });
 
       it('should do simple JSON fetch', () => {
-        return xhr.fetchJson('http://localhost:31862/get?k=v1').then(res => {
-          expect(res).to.exist;
-          expect(res['args']['k']).to.equal('v1');
-        });
+        return xhr.fetchJson('http://localhost:31862/get?k=v1')
+            .then(res => res.json())
+            .then(res => {
```

Comment:
nit: consider renaming this arg since it isn't a response, it's the result of the `json()` call